src/api/apiHelpers.ts

```diff
@@ -173,7 +173,10 @@ export async function handleGeneratedApiResponse(response: Response): Promise<an
 
   // We are not suppose to have side-effects in this function but this is a special case
   // where the access token is corrupted and we need to recreate it by logging-in again
-  if (response.status === 401 && response.statusText === 'NeedsAuthenticationResponse') {
+  if (
+    response.status === NeedsAuthenticationResponse.status &&
+    response.statusText === NeedsAuthenticationResponse.statusText
+  ) {
     eventMonitoring.captureException(new Error('NeedsAuthenticationResponse'))
```

Comment:
as-tu des raisons pour ne pas réutiliser la variable à la place de la magic string ?

```suggestion
    eventMonitoring.captureException(new Error(NeedsAuthenticationResponse.statusText))
```

Reply:
Non j'ai oublié de le modifier à cet endroit